### PR TITLE
Static UI design of JSON Viewer Page.

### PR DESCRIPTION
**Pull request by AshokPatel01**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -5,11 +5,7 @@
   "projects": {
     "json-viewer": {
       "projectType": "application",
```
Comment (mahavirvora) [inline angular.json]: keep this as it is:
```
"schematics": {
        "@schematics/angular:component": {
          "style": "scss"
        }
      },
```

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,534 +1,11 @@
...
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-
-<router-outlet></router-outlet>
\ No newline at end of file
+<header>
+  <figure class="json-viewer-logo">
+    <img src="https://saixiii.com/wp-content/uploads/2017/04/json-logo.png" alt="Json Viewer" height="40" />
```
Comment (mahavirvora) [inline src/app/app.component.html]: Don't use the logo from third parity, download it and use it.

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,534 +1,11 @@
...
+  </figure>
+  <figure class="sooryen-logo">
+    <img src="https://sooryen.com/assets/img/icons/logo-dark.svg" class="Sooryen Technologies" height="40" />
```
Comment (mahavirvora) [inline src/app/app.component.html]: Download the logo and use it locally

File: src/app/json-editor/json-editor.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { JsonEditorComponent } from './json-editor.component';
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.spec.ts]: Use absolute path

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,16 @@
...
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-  <title>JsonViewer</title>
+  <title>Json Viewer | Sooryen</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="shortcut icon" type="image/png" href="http://sooryen-dit.sooryen.com/assets/img/36x36.png" />
```
Comment (mahavirvora) [inline src/index.html]: Download it and use it internally.

File: src/styles.scss [hunk 6]
```diff
@@ -1 +1,33 @@
...
+}
+
+header {
+  background-color: #fff;
+  margin-bottom: 20px;
+  box-shadow: 0px 1px 5px RGBA(0, 0, 0, 0.1);
+  display: flex;
+  justify-content: space-between;
+}
+.json-viewer-logo {
```
Comment (mahavirvora) [inline src/styles.scss]: Move these styles inside app component scss

File: angular.json [hunk 7]
```diff
@@ -28,9 +24,13 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "./node_modules/bootstrap/dist/css/bootstrap.css",
+              "src/styles.css"
```
Comment (mahavirvora) [inline angular.json]: I can't find any styles.css inside project repo, remove it if unnecessary or change it to styles.scss

**Review by mahavirvora — CHANGES_REQUESTED** [review]
Added a couple of review comments fix those.

File: angular.json (hunk 1 above)
Comment (AshokPatel01) [reply angular.json]: Added as it is.

File: src/app/app.component.html (hunk 3 above)
Comment (AshokPatel01) [reply src/app/app.component.html]: Used it locally

File: src/app/app.component.html (hunk 2 above)
Comment (AshokPatel01) [reply src/app/app.component.html]: Used it locally

File: src/index.html (hunk 5 above)
Comment (AshokPatel01) [reply src/index.html]: Used it internally

File: src/styles.scss (hunk 6 above)
Comment (AshokPatel01) [reply src/styles.scss]: Moved CSS into app-component.scss

File: angular.json (hunk 7 above)
Comment (AshokPatel01) [reply angular.json]: Removed unused path

File: src/app/json-editor/json-editor.component.spec.ts (hunk 4 above)
Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.spec.ts]: absolute path updated.

File: package.json [hunk 8]
```diff
@@ -19,17 +19,20 @@
     "@angular/platform-browser": "~9.1.11",
     "@angular/platform-browser-dynamic": "~9.1.11",
     "@angular/router": "~9.1.11",
+    "bootstrap": "^4.5.0",
+    "jquery": "^3.5.1",
```
Comment (mahavirvora) [inline package.json]: Remove Jquery

**Review by mahavirvora — CHANGES_REQUESTED** [review]

File: src/app/app.component.html [hunk 9]
```diff
@@ -1,534 +1,11 @@
...
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-
-<router-outlet></router-outlet>
\ No newline at end of file
+<header>
+  <figure class="json-viewer-logo">
+    <img src="./assets/img/json-logo.png" alt="Json Viewer" height="40" />
```
Comment (mahavirvora) [inline src/app/app.component.html]: Don't use relative path

File: src/app/app.component.html [hunk 10]
```diff
@@ -1,534 +1,11 @@
...
+  </figure>
+  <figure class="sooryen-logo">
+    <img src="./assets/img/sooryen-logo.svg" class="Sooryen Technologies" height="40" />
```
Comment (mahavirvora) [inline src/app/app.component.html]: Don't use relative path

**Comment by AshokPatel01** [comment]
Screenshot: https://www.screencast.com/t/hrokviUTy

File: src/app/app.component.html (hunk 10 above)
Comment (AshokPatel01) [reply src/app/app.component.html]: Path Updated.

File: package.json (hunk 8 above)
Comment (AshokPatel01) [reply package.json]: Removed Jquery and unused code.

File: src/app/app.component.html (hunk 9 above)
Comment (AshokPatel01) [reply src/app/app.component.html]: path updated

File: angular.json [hunk 11]
```diff
@@ -92,9 +94,11 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "./node_modules/bootstrap/dist/css/bootstrap.css"
             ],
-            "scripts": []
+            "scripts": [
+              "./node_modules/bootstrap/dist/js/bootstrap.js"
```
Comment (mahavirvora) [inline angular.json]: Remove boottrap.js

**Review by mahavirvora — CHANGES_REQUESTED** [review]

File: angular.json (hunk 11 above)
Comment (AshokPatel01) [reply angular.json]: Removed bootstrap.js file

**Pull request merged**